restore.go

```diff
@@ -143,6 +144,20 @@ func restoreContainer(context *cli.Context, spec *specs.LinuxSpec, config *confi
 	return handler.forward(process)
 }
 
+func handleContainerstatus(status libcontainer.Status, container libcontainer.Container, context *cli.Context) (int, error) {
+	switch status {
+	case libcontainer.Running:
+		return -1, fmt.Errorf("Container with id %s already running", context.GlobalString("id"))
+	case libcontainer.Destroyed:
+		// Ensure to clean/destroy the container as the container id got created during restore startup
+		if err := container.Destroy(); err != nil {
```

Comment:
This is weird because the container state is Destoryed, then you call container.Destroy() again.  There is probably a better way to do this whole thing.